submissions/busolaAdedoyin/dom/signUp.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
You are meant to use the signup form in the html form not create another @elizabethbusola

Reply:
I only  copied and pasted the signup form in a new folder and I created a new js file alongside with it in the same folder

Reply:
There is a signup form that you have created in the htmlForm folder. Use that.

Reply:
so where should my Javascript file be because I thought we were creating a new folder for every topic

Reply:
so where should my Javascript file be because I thought we were creating a new folder for every topic

Reply:
For the dom topic, the javascript file will be in the dom folder. link the js file to the already created signup.html